xyz-hub-service/src/main/java/com/here/xyz/hub/task/FeatureTaskHandler.java

```diff
@@ -1543,11 +1545,41 @@ private static void defineGlobalSearchableField(StatisticsResponse response, Fea
     }
   }
 
-  private static void defineContentUpdatedAtField(StatisticsResponse response, FeatureTask task) {
-    StatisticsResponse.Value<Long> contentUpdatedAtVal = new StatisticsResponse.Value(task.space.contentUpdatedAt);
-    // Due to caching the value of contentUpdatedAt field could be obsolete in some edge cases
-    contentUpdatedAtVal.setEstimated(true);
-    response.setContentUpdatedAt(contentUpdatedAtVal);
+  private static Future<Void> defineContentUpdatedAtField(StatisticsResponse response, FeatureTask.GetStatistics task) {
+    Promise<Void> p = Promise.promise();
+
+    if (task.space.getExtension() != null) {
+      MultiMap queryParams = task.context.get("queryParams");
+      String spaceContextStr = queryParams.entries().stream()
+              .filter(entry -> entry.getKey().equalsIgnoreCase("context"))
+              .map(entry -> entry.getValue().toUpperCase()).findFirst().orElse(null);
+      ContextAwareEvent.SpaceContext spaceContext = ContextAwareEvent.SpaceContext.of(spaceContextStr);
+      Space.resolveSpace(task.getMarker(), task.space.getExtension().getSpaceId())
+              .onSuccess(space -> {
+                long contentUpdatedAt;
+                if (spaceContext == ContextAwareEvent.SpaceContext.SUPER) {
+                  contentUpdatedAt = space.contentUpdatedAt;
+                } else if (spaceContext == ContextAwareEvent.SpaceContext.DEFAULT) {
+                  contentUpdatedAt = space.contentUpdatedAt > task.space.contentUpdatedAt
```

Comment:
Could be simplified to: `Math.max(space.contentUpdatedAt, task.space.contentUpdatedAt)`